kernel/rros/mutex.rs

```diff
@@ -671,7 +668,7 @@ pub fn finish_mutex_wait(mutex: *mut RrosMutex) {
         if (*boosters).is_empty() {
             (*boosters).add_head((*(*mutex).next_booster).value.clone());
         } else {
-            let mut flag = 1; // flag指示是否到头
+            let mut flag = 1; // flag indicates whether to end.
```

Comment:
unclear expression

Reply:
Okay, I'll change everything that's the same.

Reply:
Can this line of comments be deleted?

Reply:
Sure.

Reply:
OK.